test/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(NOT WIN32)
 endif()
 
 add_executable(jsonbuilderTest CatchMain.cpp TestBuilder.cpp TestRenderer.cpp)
-target_compile_features(jsonbuilderTest PRIVATE cxx_std_17)
+target_compile_features(jsonbuilderTest PRIVATE cxx_std_20)
```

Comment:
>cxx_std_20 [](http://example.com/codeflow?start=48&length=10)

Out of Curiosity which C++20 the jsonbuilderTest is using. I might have missed it.

Reply:
`char8_t` and `u8string_view`.
Part of the challenge of this pr was finding a way to make jsonbuilder work nicely with these types while still allowing jsonbuilder to work with c++17.

Reply:
oh I see! for some reason I thought they were already in C++17. Good to know. 😊